### DISCUSSION TITLE
How to download

### DISCUSSION THREAD

**Discussion by krish-gaurav** [Gaming32/ArrayV#96] [Q&A] [answered]
I can't find where to download the array visualizer, I'm on Windows 64 bit computer, where and how to download? Preferably EXE or ZIP file.

**Comment by Abdullah-Ajeebi** [comment]
It's a .JAR file.

**Comment by EmeraldBlock** [comment, marked as answer]
Read the README; there's a "prebuilt JAR" that can be run with Java: https://github.com/Gaming32/ArrayV/blob/main/README.md
